libyul/optimiser/Suite.cpp

```diff
@@ -267,44 +267,29 @@ map<char, string> const& OptimiserSuite::stepAbbreviationToNameMap()
 	return lookupTable;
 }
 
+
+
 void OptimiserSuite::validateSequence(string const& _stepAbbreviations)
 {
+	stack<char> s;
+   	char ch;
```

Comment:
Please use more meaningful names. `s` and `ch` are too short and will make your code harder to follow than proper names.

See [the section on naming in our coding style doc](https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md#9-naming).